File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/ChargeInvoiceRowsInvoicesValue.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

class ChargeInvoiceRowsInvoicesValue
{
    /**
     * Unique invoice ID.
     */
    #[DTA\Data(field: "invoices_id", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "float"])]
    public float|null $invoices_id = null;

    /**
     * Description of the invoice charge.
     */
    #[DTA\Data(field: "invoices_description", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $invoices_description = null;

    /**
     * Invoice amount.
     */
    #[DTA\Data(field: "invoices_amount", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "float"])]
    public float|null $invoices_amount = null;

    /**
     * Invoice date.
     */
    #[DTA\Data(field: "invoices_date", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $invoices_date = null;

    #[DTA\Data(field: "invoices_paid", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "float"])]
    public float|null $invoices_paid = null;

    #[DTA\Data(field: "invoices_due_date", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $invoices_due_date = null;

    #[DTA\Data(field: "invoices_currency", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $invoices_currency = null;

    #[DTA\Data(field: "currency_symbol", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $currency_symbol = null;

    #[DTA\Data(field: "invoices_date_formatted", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $invoices_date_formatted = null;

    /**
     * This is optional when invoices_paid &#x3D; 1 this array will show
     */
    #[DTA\Data(field: "paid_invoices", nullable: true)]
    #[DTA\Strategy("Object", ["type" => \App\DTO\Collection17::class])]
    #[DTA\Validator("TypeCompliant", ["type" => \App\DTO\Collection17::class])]
    public \App\DTO\Collection17|null $paid_invoices = null;

}

File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/ChargeInvoiceRowsInvoicesValue.scala

package org.openapitools.client.model


case class ChargeInvoiceRowsInvoicesValue (
    /* Unique invoice ID. */
    _invoices_id: Option[Number],
    /* Description of the invoice charge. */
    _invoices_description: Option[String],
    /* Invoice amount. */
    _invoices_amount: Option[Number],
    /* Invoice date. */
    _invoices_date: Option[String],
    _invoices_paid: Option[Number],
    _invoices_due_date: Option[String],
    _invoices_currency: Option[String],
    _currency_symbol: Option[String],
    _invoices_date_formatted: Option[String],
    /* This is optional when invoices_paid = 1 this array will show */
    _paid_invoices: Option[Map[String, ChargeInvoiceRowsInvoicesValuePaidInvoicesValue]]
)
object ChargeInvoiceRowsInvoicesValue {
    def toStringBody(var_invoices_id: Object, var_invoices_description: Object, var_invoices_amount: Object, var_invoices_date: Object, var_invoices_paid: Object, var_invoices_due_date: Object, var_invoices_currency: Object, var_currency_symbol: Object, var_invoices_date_formatted: Object, var_paid_invoices: Object) =
        s"""
        | {
        | "invoices_id":$var_invoices_id,"invoices_description":$var_invoices_description,"invoices_amount":$var_invoices_amount,"invoices_date":$var_invoices_date,"invoices_paid":$var_invoices_paid,"invoices_due_date":$var_invoices_due_date,"invoices_currency":$var_invoices_currency,"currency_symbol":$var_currency_symbol,"invoices_date_formatted":$var_invoices_date_formatted,"paid_invoices":$var_paid_invoices
        | }
        """.stripMargin
}
